fcl/dunefdvd/gen/background/prodbackground_radiological_decay0_dunevd10kt_1x8x6.fcl

```diff
@@ -80,9 +50,9 @@ outputs:
    }
 }
 
-services:
-{
-    @table::services
-    @table::dunefdvd_1x8x6_3view_30deg_simulation_services
-}
-services.Geometry: @local::dunevd10kt_1x8x6_3view_30deg_v4_geo
\ No newline at end of file
+  #services.Geometry: @local::dune10kt_1x8x6_geo
+services.message.destinations.LogStandardOut.categories.BackTracker.limit: 0
+services.message.destinations.LogStandardError.categories.BackTracker.limit: 0
+
+services.message.destinations.LogStandardOut.categories.BaseRadioGen.limit: 1
+services.message.destinations.LogStandardError.categories.BaseRadioGen.limit: 1
```

Comment:
Can all of these go?

Reply:
Same as before...

Reply:
Yeah I think we should remove them and maintain a default logging policy for production jobs.

Reply:
Removing it all